src/main/java/org/janelia/saalfeldlab/n5/DataType.java

```diff
@@ -44,10 +43,15 @@ public enum DataType {
 
 	UINT8(
 			"uint8",
-			(blockSize, gridPosition, numElements) -> new ByteArrayDataBlock(
-					blockSize,
-					gridPosition,
-					new byte[numElements])),
+			(blockSize, gridPosition, numElements) -> {
+				ByteArrayDataBlock dataBlock = new ByteArrayDataBlock(blockSize, gridPosition, new byte[numElements]);
```

Comment:
this line looks like it can be deleted, and this block is stylistically different than other types.

Reply:
saw it too late, but fixed it on master now